content/applications/studio/models_modules_apps.rst

```diff
@@ -212,16 +212,87 @@ Export and import customizations
+
+.. _studio/export-import/export/configure:
+
+Configure data to export
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To select specific models to export, on the :guilabel:`Studio Export` screen click
+:guilabel:`New`, then choose the relevant model.
+
+.. tip::
+
+   Click :guilabel:`Preset` to see a list of all models in your database with records that have been
+   modifed using Studio. To configure one of these models for export, click on the model to open it
+   and make the required changes.
+
+Tick the following options as relevant:
+
+- :guilabel:`Demo`: if the exported records should be considered as demo data when imported.
+- :guilabel:`Attachments`: if attachments related to exported records should be included in the
+  exported.
+- :guilabel:`Updatable`: if the exported records should be able to updated during a module update.
 
-To import and install these customizations in another database, connect to the destination database
-and go to :menuselection:`Main dashboard --> Studio --> Customizations --> Import`, then upload
-the exported ZIP file before clicking on the :guilabel:`Import` button.
+To set or change the rules determining which of the model's records should be exported, click the
+:icon:`fa-caret-right` icon next to :guilabel:`Domain` then click :guilabel:`Edit domain`.
+
+After configuring a model for export, click :guilabel:`Studio Export` to return to the main screen.
+To download a ZIP file with the customizations for all the listed models, click :guilabel:`Export`.
+
+.. note::
+
+   It is not necessary to select one or more models as all listed models will be included in the
+   export. To remove a model from the export, select it and click
+   :icon:`fa-cog` :guilabel:`Actions` then :icon:`fa-trash-o` :guilabel:`Delete`.
+
+Choose whether to only :guilabel:`Include Data` or to also
+:guilabel:`Include Demo Data`, then click the :guilabel:`Export` button.
+
+.. image:: models_modules_apps/studio-export.png
+      :alt: Choosing to export both data and demo data
+
+When the download is complete, click the **X** to close the :guilabel:`Studio Export` window.
+
+.. _studio/export-import/import:
+
+Import customizations
+---------------------
 
 .. warning::
-   Before importing, make sure the destination database contains the same apps and modules as the
-   source database. Studio does not add the underlying modules as dependencies of the exported
-   module.
+   Before importing, make sure the destination database is the same Odoo version and contains the
+   same apps and modules as the source database. Studio does not add the underlying modules as
+   dependencies of the exported module.
+
+To import and install Studio customizations in another database, connect to the destination
+database, then, from the main Odoo dashboard click the **Toggle Studio** button then
+:guilabel:`Import`.
+
+Upload the exported ZIP file. If demo data should be imported, tick :guilabel:`Load demo data`.
```

Comment:
This could work as a bullet/numbered list:

- Connect to the DB
- Toggle studio and click Import
- Upload the exported ZIP file
- (if demo data should be imported) tick
- click install